libs/wingc/src/capture.rs

```diff
@@ -384,7 +384,12 @@ fn scan_captures_in_inflight_scope(scope: &Scope, diagnostics: &mut Diagnostics)
 			StmtKind::Class(Class {
 				methods, constructor, ..
 			}) => {
-				res.extend(scan_captures_in_inflight_scope(&constructor.statements, diagnostics));
+				if constructor.is_some() {
+					res.extend(scan_captures_in_inflight_scope(
+						&constructor.as_ref().unwrap().statements,
+						diagnostics,
+					));
+				}
```

Comment:
`if let` may be more idiomatic here, to avoid unwrap

```suggestion
				if let Some(constructor) = constructor {
					res.extend(scan_captures_in_inflight_scope(
						&constructor.statements,
						diagnostics,
					));
				}
```